### PR TITLE
Decision api Intigration

### PR DESCRIPTION
**Pull request by AbhijitP98**

### CODE REVIEW

File: src/components/molecules/DecisionCard/index.jsx [hunk 1]
```diff
@@ -57,6 +57,9 @@ const useStyles = makeStyles({
     clocklogo: {
         paddingRight: '0.2rem',
         marginBottom: '0.1rem'
+    },
+    ulist: {
```
Comment (NaveenReliable) [inline src/components/molecules/DecisionCard/index.jsx:61]: Use Proper meaning full names for class names

File: src/components/molecules/DecisionCard/index.jsx [hunk 2]
```diff
@@ -66,28 +69,29 @@ const DecisionCard = ({ data }) => {
...
                 <p className={classes.description}>Decision impact:</p>
                 <span className={classes.list}>
-                    <ul>
-                        {
+                    <ul className={classes.ulist}>
+                        {/* {
                             data.lists?.map((list, index) =>
                                 <li key={index}>{list}</li>
                             )
-                        }
```
Comment (NaveenReliable) [inline src/components/molecules/DecisionCard/index.jsx:80]: Remove commented code

File: src/components/molecules/DecisionCard/index.jsx [hunk 3]
```diff
@@ -66,28 +69,29 @@ const DecisionCard = ({ data }) => {
...
+                        } */}
+                        <li>{data.ebitdaPercentage}% in EBITDA margin.</li>
                     </ul>
                 </span>
                 <Grid className={classes.capsulGrid}>
                     <p className={classes.capsul}>
                         <img className={classes.clocklogo} src={Logo} alt="clock" loading="lazy" />
-                        {data.remainDays} DAYS AGO
```
Comment (NaveenReliable) [inline src/components/molecules/DecisionCard/index.jsx:86]: What is the issue here?

File: src/utilities/DataRequest.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ export const xhrClient = {
             headers: {
                 'accept': 'application/json',
                 'Content-Type': 'application/json',
-                'Access-Control-Allow-Origin': '*'
+                'Access-Control-Allow-Origin': '*',
```
Comment (NaveenReliable) [inline src/utilities/DataRequest.js:13]: Remove final comma mark as it is not required

**Review by NaveenReliable — CHANGES_REQUESTED** [review]

File: src/components/molecules/DecisionCard/index.jsx (hunk 1 above)
Comment (AbhijitP98) [reply src/components/molecules/DecisionCard/index.jsx]: updated with proper name

File: src/components/molecules/DecisionCard/index.jsx (hunk 2 above)
Comment (AbhijitP98) [reply src/components/molecules/DecisionCard/index.jsx:80]: removed commented codes

File: src/components/molecules/DecisionCard/index.jsx (hunk 3 above)
Comment (AbhijitP98) [reply src/components/molecules/DecisionCard/index.jsx:86]: This data not getting from api

File: src/utilities/DataRequest.js (hunk 4 above)
Comment (AbhijitP98) [reply src/utilities/DataRequest.js]: removed comma

**Pull request merged**